### PR TITLE
MongoDB 3 Contribution

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review the yml files

File: Integrations/MongoDBLog/MongoDBLog_description.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```
Comment (anara123) [inline Integrations/MongoDBLog/MongoDBLog_description.md:1]: this is README and not CHANGELOG

**Comment by Itay4** [comment]
General comments:
- add SSL
- Get incident ID youre in
- remove key-value-info
- handle list overrides
- purge-keys -> purge-entries
- add list incidents
- id, by default incident id, can also use global 
- add message arg to log
- get user automatically
- playbook is not mandatory
- entity -> id
- remove impacted
- limit read log
- read log should be table

**Comment by kirbles19** [comment]
@Shellyber Done.

File: Packs/MongoDB/Integrations/MongoDBKeyValueStore/README.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
+## Overview
+---
+
+Manipulates key/value pairs according to an incident utilizing the MongoDB collection.
+This integration was integrated and tested with version xx of MongoDB Key Value Store
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBKeyValueStore/README.md:5]: add version here
```suggestion
This integration was integrated and tested with version xx of MongoDB.
```

File: Packs/MongoDB/Integrations/MongoDBLog/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
+## Overview
+---
+
+Writes log data to a MongoDB collection.
+This integration was integrated and tested with version xx of MongoDB Log
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBLog/README.md:5]:
```suggestion
This integration was integrated and tested with version xx of MongoDB.
```

Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBLog/README.md:5]: add version

File: Packs/MongoDB/Integrations/MongoDBKeyValueStore/MongoDBKeyValueStore.py [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+        for i in incidents:
+            if i not in results:
+                results.append(i)
+    human_readable = tableToMarkdown(f'List of incidents in collecion {COLLECTION_NAME}', results,
+                                     headers=['Incidents'])
+    return human_readable, {}, {}
+
+
+def main():
+    LOG("Command being called is %s" % (demisto.command()))
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBKeyValueStore/MongoDBKeyValueStore.py:228]: use f-string

File: Packs/MongoDB/Integrations/MongoDBLog/MongoDBLog.py [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    # Point to the documents
+    cursor = COLLECTION.find({}, {'_id': False})
+    # Get count of those documents
+    log_number = cursor.count()
+    human_readable = f'The count of log documents/records is {str(log_number)}'
+    return human_readable, {}, {}
+
+
+def main():
+    LOG("Command being called is %s" % (demisto.command()))
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBLog/MongoDBLog.py:110]: use f-string

File: Packs/MongoDB/Integrations/MongoDBKeyValueStore/README.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+##### Context Output
+
+There is no context output for this command.
+
+##### Command Example
+```!mongodb-delete-key key=hello id=1234```
+
+##### Context Example
+```
+{}
```
Comment (Itay4) [inline Packs/MongoDB/Integrations/MongoDBKeyValueStore/README.md:206]: if there are not context outputs lets remove the `Context Example` section

**Pull request merged**